types/node/globals.d.ts

```diff
@@ -483,19 +483,19 @@ declare global {
             unref(): this;
         }
 
-        type TypedArray =
-            | Uint8Array
-            | Uint8ClampedArray
-            | Uint16Array
-            | Uint32Array
-            | Int8Array
-            | Int16Array
-            | Int32Array
-            | BigUint64Array
-            | BigInt64Array
-            | Float32Array
-            | Float64Array;
-        type ArrayBufferView = TypedArray | DataView;
+        type TypedArray<TArrayBuffer extends ArrayBufferLike = ArrayBufferLike> =
```

Comment:
Rather than duplicating all of `globals.d.ts` in the typesVersion dir, can we instead declare these types in a new file like `globals_typedarray.d.ts`, or even in `buffer.d.ts`?

Reply:
I guess a similar question for the differences in `buffer.d.ts` too, since it's really just one or two differing declarations.

Reply:
Awesome, that worked well for `global.typedarray.d.ts`.